src/main/java/edu/sjsu/seekers/silversnug/service/PillBoxService.java

```diff
@@ -0,0 +1,63 @@
+package edu.sjsu.seekers.silversnug.service;
+
+import edu.sjsu.seekers.silversnug.dao.AddressBookDao;
+import edu.sjsu.seekers.silversnug.dao.PillBoxDao;
+import edu.sjsu.seekers.silversnug.model.AddressBook;
+import edu.sjsu.seekers.silversnug.model.PillBox;
+import edu.sjsu.seekers.silversnug.request.PillBoxRequest;
+import edu.sjsu.seekers.silversnug.response.GenericResponse;
+import edu.sjsu.seekers.silversnug.response.PillBoxResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PillBoxService {
+
+
+    private final String SUCCESS = "SUCCESS";
```

Comment:
Can we define this in 1 single constant file?

Reply:
What do you suggest?

Reply:
define here: https://github.com/nidhijmr/SilverSnug/blob/master/src/main/java/edu/sjsu/seekers/silversnug/util/Constants.java

Reply:
Done

Reply:
@nidhijmr Please merge this so I can take latest changes